apidoc/Titanium/UI/ListView.yml

```diff
@@ -1715,7 +1715,7 @@ properties:
   - name: position
     summary: Specifies what position to scroll the selected cell to.
     type: Number
-    constants: Titanium.UI.iOS.ListViewScrollPosition.*
+    constants: [Titanium.UI.iOS.ListViewScrollPosition.*, Titanium.UI.Windows.ListViewScrollPosition.*]
```

Comment:
We should move it to Ti.UI if at least 2 platforms support it

Reply:
I think that's a very good point. I would define new constants under `Ti.UI.Windows` in this case because they may have a *same name* (such as **TOP**), but they *act differently*. 

I would think that if we can define *same behavior* across platforms for the constant, we should move it to `Ti.UI` namespace. But that was not the case for ListViewScrollPosition for now.